test/EVMHost.cpp

```diff
@@ -119,6 +119,8 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 		m_evmRevision = EVMC_SHANGHAI;
 	else if (_evmVersion == langutil::EVMVersion::cancun())
 		m_evmRevision = EVMC_CANCUN;
+	else if (_evmVersion == langutil::EVMVersion::prague())
+		m_evmRevision = EVMC_PRAGUE;
```

Comment:
I don't think this will work. The `EVMC_PRAGUE` constant won't be available until we update to an evmc version that supports it.

```suggestion
		assertThrow(false, Exception, "Prague is not supported yet.");
```

Reply:
Ah wait, looks like they already defined it the version we use: https://github.com/ethereum/solidity/blob/318e036ba489431d66b737037089ae1bf538441f/test/evmc/evmc.h#L1016-L1020

Nice.